.travis.yml

```diff
@@ -1,41 +1,34 @@
 language: php
-sudo: required
-dist: trusty
-group: edge
+sudo: false
+
 php:
+  - 5.5
   - 5.6
-  - 7.0
   - 7.1
   - 7.2
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
-
-env:
-  - SYMFONY_VERSION=2.7.*
+    - $HOME/.composer/cache
```

Comment:
you should keep caching only `$HOME/.composer/cache/files`, to avoid invalidating the Travis cache on each build due to changed packagist metadata (which will always change between builds as they change every few minutes), forcing travis to rebuild the cache archive.
thus, packages you depend on are actively developed, so they are unlikely to end up in the few files for which the cache could be a hit.

Reply:
Ok, I took this change from FOSUserBundle - should it be changed there?

Reply:
Caching only the `files` subfolder is better. but I haven't updated all repos I manage after figuring it out.